lightningd/payalgo.c

```diff
@@ -136,14 +135,7 @@ static void json_pay_sendpay_resolve(const struct sendpay_result *r,
 		return;
 	}
 
-	/* If too late anyway, fail now. */
-	if (time_after(now, pay->expiry)) {
-		/* FIXME: maybe another error kind? */
-		json_pay_failure(pay->cmd, r);
-		return;
-	}
-
-	json_pay_try(pay);
+	(void) json_pay_try(pay);
```

Comment:
Is that `(void)` needed for anything?

Reply:
Explicitly indicating we ignore the returned `bool`, but then this is not the typical style in the rest of the code...